### PR TITLE
Restore defaults of `che.factory.default_editor` property

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Restore defaults of `che.factory.default_editor` property removed in  https://github.com/eclipse/che/pull/19263
### Screenshot/screencast of this PR

Знімок екрана 2021-04-15 о 10 10 07

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/19599

### How to test this PR?
- Deploy che server image `quay.io/skabashn/che-server:che19599`
- Start che-server
- Start new workspace

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by tolusha — APPROVED** [review]

**Comment by sparkoo** [comment]
image: `quay.io/mvala/che-server:22aec6d1e839c9d0a11b3b68da5ac16a9349869c`

**Review by sparkoo — APPROVED** [review]

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties [hunk 1]
```diff
@@ -740,6 +740,9 @@ che.factory.default_plugins=NULL
 che.factory.default_devfile_filenames=devfile.yaml,.devfile.yaml
 
 ### Devfile defaults
+# Editor that will be used for factories which are created from remote git repository
```
Comment (yhontyk) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties]:
```suggestion
# Editor that will be used for factories that are created from a remote Git repository
```

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties [hunk 2]
```diff
@@ -740,6 +740,9 @@ che.factory.default_plugins=NULL
...
+# which doesn't contain any {prod-short}-specific workspace descriptor.
```
Comment (yhontyk) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties]:
```suggestion
# which does not contain any {prod-short}-specific workspace descriptor.
```

**Review by yhontyk — APPROVED** [review]

**Review by themr0c — APPROVED** [review]

**Comment by themr0c** [comment]
I apologize, my mistake.

**Pull request merged**